modules/custom/az_core/az_core.module

```diff
@@ -437,6 +439,38 @@ function az_core_form_taxonomy_overview_terms_alter(&$form, FormStateInterface $
   }
 }
 
+/**
+ * Implements hook_entity_access().
+ */
+function az_core_node_access(NodeInterface $node, $op, AccountInterface $account) {
+
+  // NetID permission is only viable for update operation.
+  if ($account->hasPermission('edit matching netid content') && ($op === 'update')) {
+    $bundle = $node->getType();
+    // We are only interested in Person content.
+    if ($bundle === 'az_person') {
+      // Determine if netid field exists.
+      if ($node->hasField('field_az_netid')) {
+        /** @var \Drupal\Core\Entity\ContentEntityBase $node */
+        $netid = $node->field_az_netid->value;
+        try {
+          // Look up user mapping for CAS.
+          $auth = \Drupal::service('externalauth.authmap')->get($account->id(), 'cas');
+          \Drupal::logger('netid_auth')->notice($auth);
```

Comment:
```suggestion
```
Is this notice intended to be a permanent addition?  (Wasn't sure if it was for debuging)